app/src/main/java/com/beemdevelopment/aegis/otp/GoogleAuthInfo.java

```diff
@@ -285,6 +288,8 @@ public Uri getUri() {
 
         if (_info instanceof YandexInfo) {
             builder.appendQueryParameter("pin", Base32.encode(((YandexInfo) _info).getPin()));
+        } else if (_info instanceof MotpInfo) {
+            builder.appendQueryParameter("pin", Base32.encode(((MotpInfo) _info).getPin()));
```

Comment:
Is there a spec for the format of QR codes?. Is the PIN supposed to be included in the QR code? And if so, is the usage of base32 encoding correct here?

Reply:
The spec agreed upon by the creators of MOTP appears to be this https://www.token2.com/?content=motpqr as referenced here http://motp.sourceforge.net/#7 and Hex encoded secrets should definitely be used, will change this